botfront/imports/ui/components/stories/common/StoryVisualEditor.jsx

```diff
@@ -166,15 +156,15 @@ export default class StoryVisualEditor extends React.Component {
     );
 
     renderSlotLine = (i, l, exceptions) => (
-        <React.Fragment key={`slot${i + l.data.name}`}>
+        <React.Fragment key={`slot${i + JSON.stringify(l.slot_was_set)}`}>
             <ExceptionWrapper exceptions={exceptions}>
-                <div className='story-line'>
-                    <SlotLabel
-                        value={l.data}
-                        onChange={v => this.handleChangeActionOrSlot('slot', i, v)}
-                    />
-                    <IconButton onClick={() => this.handleDeleteLine(i)} icon='trash' />
-                </div>
+                <SlotsContainer
+                    className={this.getReadOnlyClass()}
+                    deletable
+                    value={l.slot_was_set}
+                    onChange={slot_was_set => this.handleReplaceLine(i, { slot_was_set })}
```

Comment:
@pheel js is camelCase ;)

Reply:
These gets serialized as is, in the Rasa spec. We do similar things elsewhere when convenient, e.g. trackers, nlu models. Of course I would use camelCase otherwise. :)

Reply:
aaah alright, add an eslint comment to remove the error then